recording/src/nextcloud/talk/recording/Participant.py

```diff
@@ -202,6 +209,10 @@ def __del__(self):
             # created in "/tmp".
             self.driver.quit()
 
+    def startCommon(self):
+        if disk_usage('/tmp').free < 134217728:
+            self._logger.warning('Less than 128 MiB available in "/tmp", strange failures may occur')
```

Comment:
In general I would also prefer common code to be extracted to its own method, but in this case I think that it would be better to keep this code in `startFirefox()` and `startChrome()`.